genet/core.py

```diff
@@ -2179,3 +2180,24 @@ def write_to_csv(self, output_dir, gtfs_day='19700101'):
             persistence.ensure_dir(schedule_csv_folder)
             self.schedule.write_to_csv(schedule_csv_folder, gtfs_day)
         self.write_extras(network_csv_folder)
+
+    def add_elevation_to_nodes(self, elevation_tif_file, null_value: float):
```

Comment:
Actually you cannot have a non-float in a raster, since it is basically an image, and a pixel value in an image cannot be string - hence the weird numbers to represent null value!

Reply:
ah, gotcha, cool cool